Add a note on new repos needing a commit for resync pickup (#1680)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/installation.md

#### System Webhooks

> To create a system hook in GitLab, you must set `useSystemHook: true` and choose one of these setup methods:

Reviewer: ```suggestion
To create a system hook in GitLab, you must set `useSystemHook: true` and choose one of the following setup methods:
```

> 1. **Automatic Setup** - Provide a token with admin privileges using the `tokenMapping` parameter

Reviewer: ```suggestion
1. **Automatic Setup** - Provide a token with admin privileges using the `tokenMapping` parameter.
```

2. **Manual Setup** - Create the system hook yourself in GitLab:

>    - Follow GitLab's [system hook setup guide](https://docs.gitlab.com/ee/administration/system_hooks.html#create-a-system-hook)

Reviewer: ```suggestion
   - Follow GitLab's [system hook setup guide](https://docs.gitlab.com/ee/administration/system_hooks.html#create-a-system-hook).
```

>    - Set the URL to `{appHost}/integration/system/hook` (e.g., `https://my-gitlab-integration.com/integration/system/hook`)

Reviewer: ```suggestion
   - Set the URL to `{appHost}/integration/system/hook` (e.g., `https://my-gitlab-integration.com/integration/system/hook`).
```

>    - Enable the supported triggers: `push` and `merge_request`

Reviewer: ```suggestion
   - Enable the supported triggers: `push` and `merge_request`.
```

:::info Repository Setup and Initial Commit
Due to GitLab webhook limitations, new repositories require special handling:

> - A repository will only appear in Port after its first commit

Reviewer: the first two points are the same, fix this list

#### System Webhooks

To create a system hook in GitLab, you must set `useSystemHook: true` and choose one of the following setup methods:

1. **Automatic Setup** - Provide a token with admin privileges using the `tokenMapping` parameter.
2. **Manual Setup** - Create the system hook yourself in GitLab:
   - Follow GitLab's [system hook setup guide](https://docs.gitlab.com/ee/administration/system_hooks.html#create-a-system-hook).
   - Set the URL to `{appHost}/integration/system/hook` (e.g., `https://my-gitlab-integration.com/integration/system/hook`).
   - Enable the supported triggers: `push` and `merge_request`.

:::info Repository Setup and Initial Commit
Due to GitLab webhook limitations, new repositories require special handling:

> - A repository will only appear in Port after its first commit (GitLab webhooks don't support `project_create` events)
> - Empty repositories (no commits) will only appear after the next scheduled resync

Reviewer: ```suggestion
- A repository will only appear in Port after its first commit (GitLab webhooks don't support `project_create` events).
- Empty repositories (no commits) will only appear after the next scheduled resync.
```
